BLSQ/iaso
# IA-3444: Add missing fields to the submissions version history

Explain what problem this PR is resolving
- Add missing fields to the submissions version history
Related JIRA tickets : [IA-3444](https://bluesquare.atlassian.net/browse/IA-3444)
## Self proofreading checklist


## Doc
- Tell us where the doc can be found (docs folder, wiki, in the code...)

## Changes
- Added some more informations on the submission history like deleted status

## How to test
- Updated and delete a submission
- In the submission details go to see all version of the its history
- And check for deleted version

## Print screen / video
[Uploading Screencast from 2024-10-15 17-42-54.webm…]()



## Notes

Things that the reviewers should know: known bugs that are out of the scope of the PR, other trade-offs that were made.
If the PR depends on a PR in bluesquare-components, or merges into another PR (i.o. main), it should also be mentioned here


[IA-3444]: https://bluesquare.atlassian.net/browse/IA-3444?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
